app/assets/javascripts/home.coffee

```diff
@@ -117,15 +145,30 @@ set_error_for_invalid_fields = ->
       $('#submission_provider_down_speed').removeClass('got-error')
       $('#speed_error_span').addClass('hide')
 
+places_autocomplete = ->
+
+  placesAutocomplete = places({
+    application_id: 'pl1SUFESCKRV',
+    api_key: '6039fe8e924c5e9f9a2edd9cecba075c',
```

Comment:
Is this API key sensitive?

Reply:
API Key is a public key. According to their documentation:
> This is the public API key to use in your frontend code. This key is only usable for search queries.
I think we should store it elsewhere and reference it. Not sure how to do that.

Reply:
Thank you.